File: Practice/src/introduction_coding_tests/SumOfConsecutiveNumbers.java
package introduction_coding_tests;

import java.util.stream.IntStream;

public class SumOfConsecutiveNumbers {
//	연속된 수의 합
	public int[] solution(int num, int total) {
		return IntStream.rangeClosed(total / num - (num - 1) / 2, total / num + num / 2).toArray();
	}
}

File: Practice/src/introduction_coding_tests/SumOfConsecutiveNumbers.py
def solution(num, total):
    return [i for i in range(int(total / num - (num - 1) / 2), int(total / num + num / 2 + 1))]
    # return [(total - (num * (num - 1) // 2)) // num + i for i in range(num)]
